docs: Add Ice and Rain Protection page (#652)
flybywiresim/docs: docs/pilots-corner/advanced-guides/ice-rain-protection.md

# Ice and Rain Protection

The A320 is exposed to many types of weather and atmospheric conditions, both in flight and on the ground. These include icing conditions and heavy rain, both of which could be potentially dangerous to the aircraft if they are not managed appropriately.

> !!! tip "Icing Conditions"
>     Icing conditions on ground and during take-off can be expected when the OAT (Outside Air Temperature) is at or below 10ᵒC and there exists visible moisture on the taxiways and runways such as standing water, slush, ice, or snow which may be ingested.    
>     Icing conditions in the air can be expected when the TAT (Total Air Temperature) is at or below 10ᵒC and the aircraft is flying through visible moisture such as cloud, fog with visibility of one mile or less, rain, snow, sleet, or ice crystals.

Reviewer: ```suggestion
!!! note "Definition of Icing Conditions"
    - Icing conditions exist when the OAT (on ground or after takeoff) or the TAT (in flight) is at or below 10°C 
      and visible moisture in any form is present (such as clouds, fog with visibility of 1 sm (1 600 m) or less, rain, 
      snow, sleet or ice crystals).
    - Icing conditions also exist when the OAT on the ground and for takeoff is at or below 10°C and operating on 
      ramps, taxiways or runways where surface snow, standing water or slush may be ingested by the engines, or 
      freeze on engines, nacelles or engine sensor probes.
```

Reviewer: Aligned it with the passage I go from one of our engineers

    
To ensure safe and unrestricted operation of the aircraft in these conditions, the ice and rain protection system has been implemented. This comprises the subsystems listed below:

- [Wing Anti-Ice](#wing-anti-ice)
- [Engine Anti-Ice](#engine-anti-ice) 
- [Window Heat](#window-heat) 
- [Probes Heat](#probes-heat)
- [Rain Removal](#rain-removal)
- [Ice Detection](#ice-detection)

Jump to [Usage of the Ice and Rain Protection System](#usage-of-the-ice-and-rain-protection-system)

> ## Wing Anti-Ice
> In flight, hot air from the pneumatic system heats the three outboard slats (3-4-5) of each wing.  
> Air is supplied through one valve in each wing.  
> The WING pushbutton on the ANTI-ICE panel controls the valves.  
> When the aircraft is on ground, the flight crew can initiate a 30-second test sequence by turning the system ON.  
> If the system detects a leak during normal operation, the affected side’s wind anti-ice valve automatically closes.  
> When wing anti-ice is selected, the N1 or EPR limit is automatically reduced, and the idle N1 or EPR limit is automatically increased.  
> If the electrical power supply fails, the valves close.

Reviewer: ```suggestion
## Wing Anti-Ice

- In flight, hot air from the pneumatic system heats the three outboard slats (3-4-5) of each wing.  
- Air is supplied through one valve in each wing.  
- The WING pushbutton on the ANTI-ICE panel controls the valves.  
- When the aircraft is on ground, the flight crew can initiate a 30-second test sequence by turning the system ON.  
- If the system detects a leak during normal operation, the affected side’s wind anti-ice valve automatically closes.  
- When wing anti-ice is selected, the N1 or EPR limit is automatically reduced, and the idle N1 or EPR limit is automatically increased.  
- If the electrical power supply fails, the valves close.
```

Might be more readable with bullet points. 
Same for the other paragraphs.

If electrical power fails, the valves open.

## Window Heat
The aircraft uses electrical heating for anti-icing each windshield and demisting the cockpit side windows.  
Two independent Window Heat Computers (WHCs), one on each side, automatically regulate the system, protect it against overheating, and indicate faults.  

Window heating comes on:

- Automatically when at least one engine is running, or when the aircraft is in flight  
- Manually, before engine start, when the flight crew switches ON the PROBE/WINDOW HEAT pushbutton

Windshield heating operates at low power on the ground and at normal power in flight. The changeover is automatic.
Only one heating level exists for the windows.

## Probes Heat
Electrical heating protects:

- Pitot heads  
- Static ports  
- Angle Of Attack (AOA) probes  
- Total Air Temperature (TAT) probes  

Three independent Probe Heating Computers (PHCs) automatically control and monitor:  

- Captain probes   
- F/O probes  
- STBY probes  

They protect against overheating and indicate faults.  

> The probes are heated:  

Reviewer: ```suggestion

The probes are heated:  
```

- Automatically when at least one engine is running, or when the aircraft is in flight  
- Manually, when the flight crew switches ON the PROBE/WINDOW HEAT pushbutton  

On the ground, the TAT probes are not heated and pitot heating operates at a low level. The changeover to normal power is automatic.

## Rain Removal

### Wipers

Each front windshield has a two-speed electric wiper.
A rotary selector controls each wiper.

> Wipers must never be used on a dry windshield.

Reviewer: Also maybe bullet points again.

- The valve automatically closes if air is unavailable (engine not running).  
- When an engine anti-ice valve is open, the N1 or EPR limit is automatically reduced and, if necessary, the idle N1 or EPR limit is automatically increased for both engines to provide the required pressure.  
- If electrical power fails, the valves open.

## Window Heat  

- The aircraft uses electrical heating for anti-icing each windshield and demisting the cockpit side windows.  
- Two independent Window Heat Computers (WHCs), one on each side, automatically regulate the system, protect it against overheating, and indicate faults.  

- Window heating comes on:  

      - Automatically when at least one engine is running, or when the aircraft is in flight  
      - Manually, before engine start, when the flight crew switches ON the PROBE/WINDOW HEAT pushbutton

- Windshield heating operates at low power on the ground and at normal power in flight. The changeover is automatic.
- Only one heating level exists for the windows.

## Probes Heat  

- Electrical heating protects:  

      - Pitot heads  
      - Static ports  

>       - Angle Of Attack (AOA) probes  

Reviewer 2: ```suggestion
      - Angle of Attack (AOA) probes  
```
